serverless.yml

```diff
@@ -30,26 +33,44 @@ provider:
 plugins:
   - ./vendor/bref/bref
   - serverless-export-outputs
+  - serverless-lift
+
+constructs:
+  front:
+    type: static-website
+    path: build
+  webhook:
+    type: webhook
+    path: /webhook
+    insecure: true
+  queue:
+    type: queue
+    worker:
```

Comment:
This is here for the retry after we work out how to drop event bridge fails to the DLQ